packages/calypso-products/src/plans-list.tsx

```diff
@@ -1864,11 +1864,23 @@ export const PLANS_LIST: Record< string, Plan | JetpackPlan | WPComPlan > = {
 		getProductId: () => 2019,
 		getWhatIsIncluded: () => [
 			translate( 'Real-time backups as you edit' ),
-			translate( '1TB (1,000GB) of cloud storage' ),
-			translate( '1-year activity log archive' ),
-			translate( 'Unlimited one-click restores from the last 1 year' ),
+			translate( '{{em}}1TB (1,000GB){{/em}} of cloud storage', {
+				components: {
+					em: <em />,
+				},
+			} ),
+			translate( '{{em}}1-year{{/em}} activity log archive', {
+				components: {
+					em: <em />,
+				},
+			} ),
+			translate( 'Unlimited one-click restores from the last {{em}}1 year{{/em}}', {
+				components: {
+					em: <em />,
+				},
+			} ),
```

Comment:
It should be `<strong />`, not `<em />`

Reply:
Oops! Looks like I mistook em to be strong, while it is actually italic